### PR TITLE
Update Docstrings across the SDK

### PR DESCRIPTION
**Pull request by tribble**
## Description
I did some slight cleanup as I noticed inconsistencies.

Tagged @atainter for FGA.

Resolves DSYNC-1758

## Documentation

Does this require changes to the WorkOS Docs? E.g. the [API Reference](https://workos.com/docs/reference) or code snippets need updates.

```
[ ] Yes
```

If yes, link a related docs PR and add a docs maintainer as a reviewer. Their approval is required.

### CODE REVIEW

File: workos/audit_logs.py [hunk 1]
```diff
@@ -28,14 +40,33 @@ def create_export(
         targets: Optional[Sequence[str]] = None,
         actor_names: Optional[Sequence[str]] = None,
         actor_ids: Optional[Sequence[str]] = None,
-    ) -> AuditLogExport: ...
+    ) -> AuditLogExport:
+        """Trigger the creation of an export of audit logs.
 
-    def get_export(self, audit_log_export_id: str) -> AuditLogExport: ...
+        Kwargs:
+            organization (str): Organization's unique identifier.
```
Comment (mattgd) [inline workos/audit_logs.py:47]:
```suggestion
            organization_id (str): Organization's unique identifier.
```

File: workos/audit_logs.py [hunk 2]
```diff
@@ -28,14 +40,33 @@ def create_export(
...
+            range_start (str): Start date of the date range filter.
+            range_end (str): End date of the date range filter.
+            actions (list): Optional list of actions to filter. (Optional)
+            actors (list): Optional list of actors to filter. (Optional)
```
Comment (mattgd) [inline workos/audit_logs.py:51]:
```suggestion
            actor_names (list): Optional list of actors to filter by name. (Optional)
            actor_ids (list): Optional list of actors to filter by ID. (Optional)
```

File: workos/audit_logs.py [hunk 3]
```diff
@@ -48,13 +79,6 @@ def create_event(
         event: AuditLogEvent,
         idempotency_key: Optional[str] = None,
     ) -> None:
-        """Create an Audit Logs event.
-
-        Args:
-            organization (str) - Organization's unique identifier
-            event (AuditLogEvent) - An AuditLogEvent object
-            idempotency_key (str) - Optional idempotency key
-        """
```
Comment (mattgd) [inline workos/audit_logs.py:57]: So happy we don't have to duplicate these.

File: workos/events.py [hunk 4]
```diff
@@ -21,12 +23,24 @@ def list_events(
...
         range_start: Optional[str] = None,
         range_end: Optional[str] = None,
-    ) -> SyncOrAsync[EventsListResource]: ...
+    ) -> SyncOrAsync[EventsListResource]:
+        """Gets a list of Events.
 
+        Kwargs:
+            events (Sequence[EventType]): Filter to only return events of particular types.
+            limit (int): Maximum number of records to return. (Optional)
+            organization_id(str): Organization ID limits scope of events to a single organization. (Optional)
```
Comment (mattgd) [inline workos/events.py:32]:
```suggestion
            organization_id (str): Organization ID limits scope of events to a single organization. (Optional)
```

**Review by mattgd — APPROVED** [review]
Thanks for doing this! A few minor suggested edits.

File: workos/mfa.py [hunk 5]
```diff
@@ -22,26 +22,83 @@
...
-    ) -> AuthenticationFactorExtended: ...
+    ) -> AuthenticationFactorExtended:
+        """
+        Defines the type of MFA authorization factor to be used. Possible values are sms or totp.
+
+        Kwargs:
+            type (str): The type of factor to be enrolled (sms or totp).
+            totp_issuer (str): Name of the Organization. (Optional)
+            totp_user (str): email of user. (Optional)
+            phone_number (str): phone number of the user. (Optional)
```
[the remark this answers is not in the document]
Comment (tribble) [reply workos/mfa.py:42]: Good call. I'm going with `Required when type is totp, ignored otherwise.` and removing the `(Optional)`

**Pull request merged**